source/client/ClientState.py

```diff
@@ -8,22 +8,30 @@ class ClientState:
     It stores what is needed to compute scores and decide on move legality
     """
 
-    def __init__(self, ruleset = None):
+    def __init__(self, ruleset = 'tbd'):
         """Initialize a state tracker for a given client"""
-        if ruleset is not None:
-            rule_module = "common." + ruleset
-        else:
-            #This is the unit test case - we may want to put a dummy ruleset in
-            print("In unittest mode - using HandAndFoot rules")
-            rule_module = "common.HandAndFoot"
+        self.ruleset = ruleset
```

Comment:
This was a pre-existing thing, but we probably don't need to keep self.ruleset or require it on init, since we are passing it to the importRuleset method.

Reply:
With current architecture, Tableview needs to be passed the ruleset.  
I was able to eliminate clientState.rule_module (rule_module is now purely local to importRuleset method).

Reply:
I did eliminate the duplicate code.